packages/create-cloudflare/src/frameworks/angular/index.ts

```diff
@@ -3,24 +3,24 @@ import { resolve } from "node:path";
 import { logRaw } from "helpers/cli";
 import { brandColor, dim } from "helpers/colors";
 import {
-	detectPackageManager,
 	installPackages,
 	runCommand,
 	runFrameworkGenerator,
 } from "helpers/command";
 import { readFile, readJSON, writeFile } from "helpers/files";
 import { spinner } from "helpers/interactive";
+import { detectPackageManager } from "helpers/packages";
 import { getFrameworkVersion } from "../index";
 import type { PagesGeneratorContext, FrameworkConfig } from "types";
 
-const { npx, npm } = detectPackageManager();
+const { dlx, npx, npm } = detectPackageManager();
```

Comment:
Why is `npx` still here? Why are only some of the `npx`s changed to `dlx`?

Reply:
Yeah, specifically just the bits where we invoke the framework scaffolding CLIs. `dlx` is designed for those situations where you aren't in a package context and don't want to invoke something from node-modules